From b7996852a5e3c46ad1df041de5113e442cabcc6d Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Tue, 7 Mar 2023 15:39:29 +0100
Subject: [PATCH 430/529] ARM: configs: enable JFFS2_FS on
 fragment-02-multiv7_addons

Enable JFFS2 file system on multiv7 addons fragment.

Change-Id: I64046590a027aec8a8f1db9dbde8a10da59664ab
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/294207
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -36,6 +36,7 @@ CONFIG_HWSPINLOCK=y
 CONFIG_HWSPINLOCK_STM32=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
+CONFIG_JFFS2_FS=y
 CONFIG_KEYS=y
 CONFIG_LEGACY_PTY_COUNT=8
 CONFIG_LOG_BUF_SHIFT=16
